templates/admin/choose-engine.php

```diff
@@ -0,0 +1,119 @@
+<div class="wrap">
+
+    <?php include('common/header.php'); ?>
+
+    <div class="swiftype-admin">
+        <div class="main-content">
+            <h3><?= __('Let\'s create an engine !'); ?></h3>
```

Comment:
From a product perspective, I think it would be good to have some sort of description of what an engine actually is. If you're a wordpress user who has no idea what Swiftype is, you just want to add a search box to your site, you might wonder here.

Reply:
@goodroot: Do you have any description of what an engine is somewhere or should I try to write mine?

Reply:
@afoucret It would be great if you could provide your own.

Any changes made to language will be done "at once", to ensure everything is consistent. 👍

Reply:
Have added something for this:
Let you review again.